### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
-> The repository is in preview mode. Only Angular version is available for use. Vue and React projects are still under development.
+## Get Started
+
+To run these UI templates on your machine, clone this repository and follow the instructions below.
+
+1. Run the `npm i` command in the framework folder to install packages to the project and prepare the repository. 
+
+2. Start the project with one of the following commands depending on the framework of your choice:
+
+    - `ng serve` for Angular
```
Comment (groshenkovamarina) [inline README.md:25]: ng serve => npm start

Comment (vladaskorohodova) [inline README.md:25]:
```suggestion
    - `npm start` for Angular
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
+Use the left menu to navigate through the CRM, Planning and Analytics views.
 
 The UI Templates have responsive layouts with DevExtreme UI components. Note that the components use sample data entities without any business logic. You can use these UI Templates in your project and adapt them to your specific business requirements.
 
-> The repository is in preview mode. Only Angular version is available for use. Vue and React projects are still under development.
+## Get Started
+
+To run these UI templates on your machine, clone this repository and follow the instructions below.
+
+1. Run the `npm i` command in the framework folder to install packages to the project and prepare the repository.
```
Comment (helen-dikareva) [inline README.md:21]: I think the first step should be
1. Choose a corresponding framework folder and go there
React - `cd packages\react`
Angular - `cd packages\angular`
Vue - `cd packages\vue`
2. Run the `npm i` command to install packages to the project.
...

On mac of course there are different slashes, we can give paths for both Windows and Mac but I think mac users know what to do :-)

Comment (vladaskorohodova) [reply README.md]: Noted

File: README.md [hunk 3]
```diff
@@ -1,21 +1,48 @@
...
+Use the left menu to navigate through the CRM, Planning and Analytics views.
 
 The UI Templates have responsive layouts with DevExtreme UI components. Note that the components use sample data entities without any business logic. You can use these UI Templates in your project and adapt them to your specific business requirements.
 
-> The repository is in preview mode. Only Angular version is available for use. Vue and React projects are still under development.
+## Get Started
+
+To run these UI templates on your machine, clone this repository and follow the instructions below.
+
+1. Choose a framework and go to a corresponding folder:
```
Comment (helen-dikareva) [inline README.md:21]: @vladaskorohodova  Sorry, my bad.
I've just looked carefully and realized that I wrote it incorrectly
The actual step sequence is
1. Install modules with `npm i` command
2. Go to a desired framework folder
3. Call the run script
4.  Open the application on the local server. The link will be in the console output.

File: README.md [hunk 4]
```diff
@@ -16,27 +16,33 @@ The UI Templates have responsive layouts with DevExtreme UI components. Note tha
...
+To run these UI templates on your machine, clone this repository and run the following commands in the root folder:
 
-1. Choose a framework and go to a corresponding folder:
+### Angular
 
-    - `cd packages\angular` for Angular
+    ```
+    npm install
+    cd packages\angular
+    npm run start
```
Comment (helen-dikareva) [inline README.md:26]: It can be just `npm start`

**Review by helen-dikareva — APPROVED** [review]

**Review by dxbykov — CHANGES_REQUESTED** [review]
I think there is no reason to replicate the UI Gallery navigation structure in this readme. It will be hard to maintain (we'll need to do all such changes twice). I would write this readme in a way so we don't need to change it each time we introduce new templates.

File: README.md [hunk 5]
```diff
@@ -1,54 +1,55 @@
...
-The UI Templates have responsive layouts with DevExtreme UI components. Note that the components use sample data entities without any business logic. You can use these UI Templates in your project and adapt them to your specific business requirements.
-
-> The repository is in preview mode. Only Angular version is available for use. Vue and React projects are still under development.
-
-![Devextreme UI Template Gallery](/images/ui-template-gallery.png)
-
-## How It Organized
-
-UI Template sources for Angular are in the [packages/angular](packages/angular) folder. Navigate to the following folders to get the code for each template:
+Take a look at the gallery **online** here:
```
Comment (dirk-pieterse) [inline README.md:7]:
```suggestion
Take a look at the **online** gallery here:
```

**Review by dirk-pieterse — APPROVED** [review]

**Review by dxbykov — APPROVED** [review]

**Pull request merged**